cpp/include/cudf/strings/strings_column_view.hpp

```diff
@@ -78,6 +99,28 @@ class strings_column_view : private column_view {
    * view  (i.e.: non-zero offset or reduced row count).
    */
   size_type chars_size() const noexcept;
+
+  /**
+   * @brief Return an iterator for the chars child column.
+   *
+   * This does not applies the offset of the parent.
+   * The offsets child must be used to properly address the char bytes.
+   *
+   * For example, to access the first character of string `i` (accounting for
+   * a sliced column offset) use: `chars_begin()[offsets_begin()[i]]`.
+   *
+   * @return Iterator pointing to the first char byte.
+   */
+  chars_iterator chars_begin() const;
+
+  /**
+   * @brief Return an end iterator for the offsets child column.
+   *
+   * This automatically applies the offset of the parent.
```

Comment:
Why does chars_end apply the offset of the parent if `chars_begin()` does not?  I think maybe this is just a copy paste error in the docs?